domain/src/main/java/ru/startandroid/domain/models/placesmodels/citiesdetailsmodels/AddressComponents.kt

```diff
@@ -0,0 +1,9 @@
+package ru.startandroid.domain.models.placesmodels.citiesdetailsmodels
+
+
+data class AddressComponents (
+
+	val long_name : String,
```

Comment:
Name of members doesn't follow java code convention. The same in some other classes.

Reply:
Deleted